climada/hazard/tc_tracks.py

```diff
@@ -1428,7 +1428,7 @@ def _one_interp_data(track, time_step_h, land_geom=None):
             method = ['linear', 'quadratic', 'cubic'][min(2, track.time.size - 2)]
 
             # handle change of sign in longitude
-            lon = track.lon.copy()
+            lon = u_coord.lon_normalize(track.lon.copy(), center=0)
```

Comment:
The normalization step introduces a little computational overhead, but I'm quite sure that it's neglectable compared to the other operations in this function.

Reply:
worth it

Reply:
would it be worth normalising the longitudes when we read in the tracks instead of when we interpolate?

we already make one adjustment to longitude when reading in (tc_tracks.py lines 527-534), and this way we can be more confident that CLIMADA will 'play nice' with all the tracks.

do you know what the overhead is to normalise longitudes for the full event set? it looks like it would still be quite cheap?

Reply:
thanks, and agreed